app/assets/stylesheets/pdf/_patternfly.scss.erb

```diff
@@ -4,7 +4,7 @@
 
 @font-face {
   font-family: "PatternFlyIcons-webfont";
-  src: url(<%= asset_data_uri("bower_components/patternfly/dist/fonts/PatternFlyIcons-webfont.ttf") %>);
+  src: url(<%= asset_data_uri("patternfly/PatternFlyIcons-webfont.ttf") %>);
```

Comment:
Probably not .. this matches what @epwinchell removed in https://github.com/ManageIQ/manageiq-ui-classic/pull/3041

Reply:
Probably `patternfly-sass/dist/fonts/PatternFlyIcons-webfont.ttf` (at least according to the actual path in the gem).

Reply:
Aah... @skateman...

```
assets_environment.paths
=> [
...
 "/home/himdel/.rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/patternfly-sass-3.38.2/dist/fonts",
...
]
```

```
assets_environment.find_asset('patternfly/PatternFlyIcons-webfont.ttf')                                                                  
=> nil

assets_environment.find_asset('PatternFlyIcons-webfont.ttf')
=> #<Sprockets::Asset:3f9f4815d19c "file:///home/himdel/.rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/gems/patternfly-sass-3.38.2/dist/fonts/PatternFlyIcons-webfont.ttf?type=application/x-font-ttf&id=5055203a3ed21d2f1c79194f2d7d9550074f803c14537a47e35d9378e7f17398">
```

So... this should just be `asset_data_uri("PatternFlyIcons-webfont.ttf")`

Reply:
I agree with @himdel ;)

Reply:
`asset_path`

Reply:
But I still don't know on what screen I can test this? @himdel or @hstastna

Reply:
oh guys, this file is terrible, I will rewrite it to SASS (yes we can do it) :wink: in a next PR